### DISCUSSION TITLE
Set custom class to active button

### DISCUSSION THREAD

**Discussion by vprint** [quasarframework/quasar#16077] [General - Components / Directives / etc]
Here is an extract of a component i'm working on:
```
<div v-for="layer in backgroundLayers" :key="layer.NAME" class="regular-div">
  <div class="col-3 col-sm-10 button-center">
    <q-btn round class="q-ma-xs" @click="setLayer(layer.NAME)">
      <q-avatar size=" 70px">
        <img v-if="layer.TOKEN" :src="`${layer.IMG}access-token=${layer.TOKEN}`">
        <img v-else :src="`${layer.IMG}`">
      </q-avatar>
    </q-btn>
  </div>
  <div class="col-3 col-sm-2">
    <p class="regular-font q-ma-xs">{{ layer.NAME }}</p>
  </div>
</div>
```

with the following css : 
```
.round-button
  margin: 2px

.round-button:active
  margin: 0px
  border: 2px solid $primary
```

This give me this :
<img width="232" alt="image" src="https://github.com/quasarframework/quasar/assets/42189444/80c0870f-1f54-4e7e-9fc9-5381586c60b7">

I want to set the .round-button class when layer.NAME === 'OSM' so i've tried to add this code :class="{ active: layer.NAME === 'OSM' }". But this is not working.

I'm expecting this result :
<img width="230" alt="image" src="https://github.com/quasarframework/quasar/assets/42189444/a745bf91-2a1c-4823-8faa-1ec709410a55">

Any idea on how can i achieve that ?

Valentin

**Comment by suleimanas** [comment]
do it like this:
`:class="[layer.NAME === 'OSM' ? 'active' : '']"`
